lib/evergreen.js

```diff
@@ -63,10 +70,37 @@ function zipPlatform (projectPath, platform) {
   return zip(srcPath, zipPath);
 }
 
+// assumeAWSRole () => Promise<Object>
+function assumeAWSRole () {
+  return blinkMobileIdentity.assumeAWSRole();
+}
+
+// assumeAWSRole () => Promise<String>
+function getCurrentTenant () {
+  return blinkMobileIdentity.getTenants()
+    .then((tenants) => {
+      if (!tenants || !tenants.current) {
+        return Promise.reject(new Error('Current tenant is not set, please use \'bm identity tenant\' command to set a tenant before continuing.'));
```

Comment:
I did think about this, however, decided against it as if this was being used in CI context, you wouldnt want a prompt to start. When there are few things in the CLI that i would like to check if the its running in interactive mode (apparently this is doable??) and prompt rather than error out. Checking if the user is logged in before trying to assume an AWS role is another example where it currently just errors.